docs/Compliance.md

```diff
@@ -0,0 +1,124 @@
+# ATO and compliance considerations
+
+Every federal information system must be granted an Authority To Operate (ATO)
+by an Authorizing Official in order to go into production.  A good overview of
+the process can be found in this [excellent ATO overview](https://docs.google.com/presentation/d/1x-Bt8uyW-szHarglY57fFcC6PRCpUGFLe07uOvt93uk/edit#slide=id.g1f710bd7ce_0_847)
```

Comment:
This doc wouldn't be to be viewed by vendors or the outside. Perhaps we should remove or point to a public resource.